### ISSUE TITLE
Crashing every 30 minutes

### ISSUE THREAD

**Issue by nyxriella** [KurtBestor/Hitomi-Downloader#6192]
Anyone have the same problem?
I don't know how or where to check the error/log.

**Comment by KurtBestor** [comment]
Run with `hitomi_downloader_GUI.exe --debug`
Please post the `debug.txt`

**Comment by tatuta630** [comment]
We are currently using the 3.8d technical preview.
It crashes shortly after starting.
I have attached debug.txt.
[debug.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/12147640/debug.txt)

**Comment by KurtBestor** [comment]
Is it the same in [v3.8d](https://github.com/KurtBestor/Hitomi-Downloader/releases/tag/v3.8d) and [v3.8c](https://github.com/KurtBestor/Hitomi-Downloader/releases/tag/v3.8c)?

**Comment by tatuta630** [comment]
No, I only became aware of this issue a week ago, so I'm assuming it's probably "3.8d" only.

**Comment by tatuta630** [comment]
I just updated to v3.8e, but the symptoms continue to occur.
[debug.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/12157489/debug.txt)

**Comment by KurtBestor** [comment]
Press <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>C</kbd> to open the console and type the following command and press <kbd>Enter</kbd>:
```python
sys.getrecursionlimit()
```
Show me the output

**Comment by tatuta630** [comment]
I got a crash dump when it crashed.
Upload the result analyzed by Windbg.
[analyzed.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/12257434/analyzed.txt)

**Comment by KurtBestor** [comment]
What GPU are you using?

**Comment by tatuta630** [comment]
I'm using RTX 2060 SUPER (Mem:6GB).

**Comment by KurtBestor** [comment]
Run the program with the command below:
```
hitomi_downloader_GUI.exe --disable-gpu
```
Do you still have the same symptoms?

**Comment by tatuta630** [comment]
Unfortunately, it crashed.
